packages/world-local/src/queue.ts

```diff
@@ -2,6 +2,7 @@ import { setTimeout } from 'node:timers/promises';
 import { JsonTransport } from '@vercel/queue';
 import { MessageId, type Queue, ValidQueueName } from '@workflow/world';
 import { monotonicFactory } from 'ulid';
+import { Agent } from 'undici';
```

Comment:
A little sad we need a 1.5MB package just to set a timeout, but I also don't see a better workaround here

Reply:
Yep. Pretty lame that Node doesn't export this itself.